.env.example

```
@@ -5,3 +5,4 @@ PERPS_API_KEY=
 PERPS_API_URL=https://perps.yield.xyz
 YIELDS_API_KEY=
 YIELDS_API_URL=https://api.yield.xyz
+#YIELD_ID= optional quick test mode
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)